### ISSUE TITLE
Having a NetworkManager symlinked resolv.conf results in empty symlinks in the container.

### ISSUE THREAD

**Issue by hdonnay** [debarshiray/toolbox#231]
It ends up with `/etc/resolv.conf` linked to `/run/host/etc/resolv.conf/` linked to `/var/run/NetworkManager/resolv.conf`, which doesn't exist inside the container.

This can be fixed on the host system by doing something like the following:

```sh
rm /etc/resolv.conf && cp /var/run/NetworkManager/resolv.conf /etc/resolv.conf
```

This will then cause NetworkManager to update the file directly instead of updating its copy in `/run`. (Documented [here](https://developer.gnome.org/NetworkManager/stable/NetworkManager.conf.html), look for `rc-manager`)

---

Some sort of warning that `/etc/resolv.conf` is a dangling symlink would help, as the default toolbox container is missing `dig` or `nslookup` and `ping` seemingly doesn't work.

**Comment by debarshiray** [comment]
Interesting. Which distribution is this?

This is essentially a duplicate of https://github.com/debarshiray/toolbox/issues/187

**Comment by hdonnay** [comment]
Fedora 30 with NetworkManager.

I don't recall if it's a stock thing or if I pushed it into the symlink-managed thing.

**Comment by debarshiray** [comment]
I see that `/run/NetworkManager/resolv.conf` does exist on Fedoras 29 and 30. However, `/etc/resolv.conf` isn't a symbolic link to it.

**Comment by hdonnay** [comment]
I guess it's something I did, then, but it does seem to be a supported NetworkManager config.

Would you be open to a patch that tries to detect if things are configured that way and either prints a warning or just copies the file?

**Comment by debarshiray** [comment]
I taught myself a bit more about NetworkManager and it's handling of `resolv.conf`, so I understand things a bit better now. I think you either changed `main.rc-manager` in `NetworkManager.conf` or you set up the symbolic link yourself.

Either way, it doesn't matter for the purposes of this issue. This is a duplicate of #187 

We already have a known way to fix this problem, but it requires Flatpak 1.4.0, which isn't present on Fedora 29. So we are waiting for Fedora 29 to reach its end-of-life before we deploy the fix.

**Comment by debarshiray** [comment]
Duplicate of #187
